Documentation/CRDs/Cluster/network-providers.md

```diff
@@ -418,6 +418,23 @@ spec:
     }'
 ```
 
+## Holder Pod Deprecation
+
+Rook plans to deprecate CSI "holder" pods in Rook v1.16. Clusters with `csi-*plugin-holder-*` pods
+present in the Rook operator namespace must plan to set `CSI_DISABLE_HOLDER_PODS` to `"true"` after
+Rook v1.14 is installed and before v1.16 is installed by following the migration sections below.
+
+CephClusters that do not use `network.provider: multus` can skip to the
```

Comment:
They can skip the section completely, right?
```suggestion
CephClusters that do not use `network.provider: multus` can skip the
```

Reply:
"skip to" is what I intended, but I can see this might be confused easily, so I changed it to "follow"